### PR TITLE
Fixes issue #34 and #36:Added option for adding emoji icon.

### PR DESCRIPTION
**Pull request by amitiwary999**
Fixes issue #34 and #36 When user will click on image icon (left one) present in edittext emoji icons will be popup so that user can insert emoji icons also and by this it also solved the issue #34.

### CODE REVIEW

**Comment by betterclever** [comment]
@amitiwary999 Please go through my review in your previous Pull Request. I highlighted some points there that are missing in this PR too. Thanks.

**Comment by amitiwary999** [comment]
Thanks for your review but I added all the points you mentioned in your review.

**Comment by shivenmian** [comment]
Squash your commits. @amitiwary999

File: build.gradle [hunk 1]
```diff
@@ -7,7 +7,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.0'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```
Comment (betterclever) [inline build.gradle]: The version used should be 2.2.0 .

Comment (betterclever) [inline build.gradle]: I mentioned this earlier too.

Comment (betterclever) [inline build.gradle]: I read your comment in previous PR. Are you using Android Studio 2.1.3? This plugin is compatible with 2.2.0 maybe that is the cause for the error.

Comment (amitiwary999) [inline build.gradle]: Yes I'm using 2.1.3.

**Comment by shivenmian** [comment]
Squash your commits @amitiwary999

File: build.gradle (hunk 1 above)
Comment (gogeta95) [inline build.gradle]: @amitiwary999 update yours to 2.2 for bug fixes and new features ASAP.

**Pull request closed without merging**